app/src/org/commcare/activities/CommCareHomeActivity.java

```diff
@@ -1160,12 +1162,15 @@ public void onClick(DialogInterface dialog, int i) {
         showAlertDialog(d);
     }
 
-    void displayMessage(String message) {
-        displayMessage(message, false);
+    @Override
+    public void reportSuccess(String message) {
+        displayMessage(message, false, false);
     }
 
-    void displayMessage(String message, boolean bad) {
-        displayMessage(message, bad, false);
+
+    @Override
+    public void reportFailure(String message, boolean showPopupNotification) {
```

Comment:
It is actually used in https://github.com/dimagi/commcare-odk/pull/1059 in some changes that came up during resolving a merge conflict that maybe should have been pulled into here as well...